BitPayLib/class-bitpayipnprocess.php

```diff
@@ -39,43 +42,64 @@ public function __construct(
 	public function execute( WP_REST_Request $request ): void {
 		$data = $request->get_body();
 
-		$data       = json_decode( $data, true, 512, JSON_THROW_ON_ERROR );
-		$event      = $data['event'] ?? null;
-		$data       = $data['data'] ?? null;
-		$invoice_id = $data['id'] ?? null;
+		$data                = json_decode( $data, true, 512, JSON_THROW_ON_ERROR );
+		$event               = $data['event'] ?? null;
+		$data                = $data['data'] ?? null;
+		$data['event']       = $event;
```

Comment:
@mwarzybok-sumoheavy do we need to reassign `$data['event']` here?

Reply:
yes, cause like you see we override $data (line 45 and 47) so it doesn't contain 'event' in structure. I didn't want to change structure from past (backward compatibility), just only add two fields. Of course we can just put whole json to the logger. It depends on you.